### ISSUE TITLE
Docs: Is this only useful when paying with Monzo directly (not Oyster)?

### ISSUE THREAD

**Issue by OJFord** [jameshill/mondo-tfl#4]
There are objective advantages to paying with Oyster in some cases:
- Travelcards outside Zone 1 or for >=1mo duration
- Railcard holders (with it linked to Oyster)
- Students
- Other concessions

However, the top-up or travelcard payment might still come from Monzo. In this case, will journey information be linked to the Monzo transactions, or does this project only handle paying with a Monzo debit card directly per-journey?

**Comment by jameshill** [comment]
yep, only useful when paying directly with Monzo. Good idea though!
